packages/@coorpacademy-components/src/molecule/brand-tabs/index.js

```diff
@@ -18,13 +19,19 @@ const Tab = ({tab, index, isLightTab}) => {
       onClick={onClick}
     >
       <Link href={href}>{title}</Link>
+      {!isUndefined(status) ? (
```

Comment:
si tu utilise  !isUndefined tu permet l'utilisation de chaines vide, dans ce cas t'aura une bulle vide :/  
```suggestion
      {!status? (
```

Reply:
c'est plutôt `status ?`